.pre-commit-config.yaml

```diff
@@ -1,6 +1,15 @@
--   repo: git@github.com:Yelp/detect-secrets
-    rev: 0.10.1
+repos:
+  - repo: git@github.com:Yelp/detect-secrets
+    rev: v0.13.0
     hooks:
-    -   id: detect-secrets
-        args: ['--baseline', '.secrets.baseline']
+      - id: detect-secrets
+        args: ["--baseline", ".secrets.baseline"]
         exclude: .*/build/.*|.*/test/.*|.*/androidTest/.*
+  - repo: local
```

Comment:
You can actually use @macisamuele 's KtLint wrapper:

```
- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
  rev: v1.3.2
  hooks:
  - id: pretty-format-kotlin
    args: [--autofix]
```

As we did for `swagger-gradle-codegen`:

https://github.com/Yelp/swagger-gradle-codegen/blob/master/.pre-commit-config.yaml#L14-L20

Reply:
Where is the fun in that? (Though, good idea)